packages/schemas/resources/css/components/wizard.css

```diff
@@ -2,11 +2,15 @@
     @apply flex flex-col;
 
     & .fi-sc-wizard-header {
-        @apply grid divide-y divide-gray-200 md:grid-flow-col md:divide-y-0 md:overflow-x-auto dark:divide-white/5;
+        @apply grid divide-gray-200 md:grid-flow-col md:divide-y-0 md:overflow-x-auto dark:divide-white/5;
```

Comment:
Adding this class back doesn't hurt anything, right? I am just thinking if a user wanted to disable this behaviour by overriding `&:not(.fi-active) {` in their own stylesheet, they wouldn't also need to redefine this.

```suggestion
        @apply grid divide-y divide-gray-200 md:grid-flow-col md:divide-y-0 md:overflow-x-auto dark:divide-white/5;
```

Reply:
When it's there, there's an extra line under the header on mobile.

![image](https://github.com/user-attachments/assets/16d0aa57-a78d-4b41-9dc3-556e4cd6ed56)

Reply:
Okay, please remove all other divide classes from that line then.

Reply:
Done